script/collect.js

```diff
@@ -31,11 +31,13 @@ async function getDocsForNodeVersion (major, version) {
 
   // move docs from temp dir to this repo
   const tempDocDir = path.join(tempDir, `node-${version.replace('v', '')}`, 'doc')
-  await fs.move(tempDocDir, docDir)
-  fs.remove(tempDir)
 
-  // keep .md files and remove all others
-  walk(docDir, {directories: false})
-    .filter(file => path.extname(file.relativePath.toLowerCase()) !== '.md')
-    .forEach(file => fs.unlinkSync(path.join(docDir, file.relativePath)))
+  // removes files other than markdown
+  walk(tempDocDir, {directories: false})
+  .filter(file => path.extname(file.relativePath.toLowerCase()) !== '.md')
+  .forEach(file => fs.unlinkSync(path.join(tempDocDir, file.relativePath)))
+  
+  await fs.copy(tempDocDir, docDir)
```

Comment:
Can you explain the reasoning behind changing the order of things here?

Reply:
simply we don't have to copy all the content in the docsDir and delete them afterwards, rather we can filter the docs in the tempDir and copy relevant (only markdown) to the docsDir

deletion will take same time but the time to copy will be reduced